vnext/Microsoft.ReactNative/IViewManager.idl

```diff
@@ -3,14 +3,28 @@
 
 namespace Microsoft.ReactNative.Bridge
 {
+	[webhosthidden]
+	enum ViewManagerPropertyType
```

Comment:
>ViewManagerPropertyType [](start = 6, length = 23)

It would be nice if we do not have such enum. 
Ideally developers should have any type they like, and we provide conversions based on IJSValueReader/IJSValueWriter.